bin/generate_unfiltered_sce.R

```diff
@@ -197,6 +197,8 @@ sample_type <- sample_metadata_df |>
   dplyr::filter(sample_id %in% sample_ids) |>
   dplyr::mutate(
     sample_type = dplyr::case_when(
+      # account for sample being both cell line and PDX
+      is_xenograft & is_cell_line ~ "cell line, patient-derived xenograft",
```

Comment:
I am wary of two character splits, even though I know this works.
```suggestion
      is_xenograft & is_cell_line ~ "cell line;patient-derived xenograft",
```